MIP/mip-71/README.md

```diff
@@ -0,0 +1,102 @@
+
+
+Since the Relayer may use a confirmation level that is not the finalization, we add the following requirement:
+
+6. The confirmation levels for the Informer MUST be the same as for the Relayer.
+
+#### Locked and minted supply difference
+
+The Informer MUST provide the information about the difference of locked \$L1MOVE and minted \$L2MOVE supply `MOVE_diff`. This indirectly informs, whether there is an issue with the total circulating supply of \$MOVE across the L1 and L2. The Informer provides this information at time `t'`, where `t' ~ max{t_L1,t_L2}`). It is approximate, since the local time of the informer may differ from the chain clocks. However, it can only provide guarantees for `t = min {t_L1,t_L2}`, which is discrete block time.
+
+The Informer MUST measure `MOVE_diff(t) = L1MOVE_locked(t_L1) - L2MOVE_mint(t_L2)` and the condition `is_MOVE_diff_safe = (MOVE_diff < 0)`.
+
+> [!NOTE]
+`CONDITION_1` :  The Informer warns if `is_MOVE_DIFF` is false. This is because in this scenario the circulating supply exceeds the maximum circulating supply `MOVE_Max`.
+
+#### Inflight tokens
+
+The Informer MAY calculate the in-flight tokens. Inflight tokens are tokens that are locked (burned) on L1 (L2) but that are not yet minted (unlocked) on L2 (L1) with a **finalized state**. With respect to finalized states it should be `MOVE_inflight = MOVE_diff`.
+
+We can further differentiate the inflight tokens, however this considers non-finalized events and thus results may not be precise or may even revert.
+
+We can differentiate the following types of inflight tokens:
+
+1. transfer is processed by the Relayer and the `complete_transfer` transaction is recorded on the target chain but not finalized. This includes the sets `L1MOVE_inflight_processing` and `L2MOVE_inflight_processing`.
+1. transfer is initiated on source chain but the Relayer has not yet processed the transfer. This includes the sets `L1MOVE_inflight_init` and `L2MOVE_inflight_init`.
+
+The Informer can measure `L1MOVE_inflight_processing` and `L2MOVE_inflight_processing` by reading the non-finalized state of the respective chain.
+
+## Recommendations
+
+#### Possible measurement errors
+
+The timestamps of the two layers are not synchronized. We assume that the difference is negligible, however, for correctness, the implications of a drift between the L1 and L2 clocks should be considered. Moreover, the clocks of the layers progress discretely. This means that the Informer reads events from L1 and L2 with slightly different time stamps. This timestamp difference can introduce errors in the calculation of circulating token.
+
+It is recommended that the Informer SHOULD not rely on timestamps.
```

Comment:
fair point.. i removed the sentence

Reply:
i think this is where postconfirmations may come into play, which forces some sort of synchronization. Essentially since L1-finalized postconfirmations take O(12min), transfers also increase in time for L1->L2 transfers. It is fair to assume that with such large time windows the clocks are sufficiently close. The only insecurity arises for inflight tokens which is not relevant for Informer V1.  Long term we will study the implications. @franck44 may be following up on this and look into it using UPPAAL